src/__tests__/fixtures/regression.css

```diff
@@ -0,0 +1,7 @@
+@custom-selector :--button .button;
+
+:--button {
+  &--active {
```

Comment:
@tabatkins sorry that took me so long I just wanted to triple check as @MoOx asked.